bin/desi_qso_qn_afterburner

```diff
@@ -359,11 +359,20 @@ if __name__ == "__main__":
     ## Selection param for QuasarNet
     param_QN = {'c_thresh':args.c_thresh, 'n_thresh':args.n_thresh}
 
-    param_RR = {'template_filename':args.template,
-                'filename_priors':args.filename_priors,
-                'filename_output_rerun_RR':args.filename_output_rerun_RR,
-                'filename_redrock_rerun_RR':args.filename_redrock_rerun_RR,
-                'delete_RR_output':args.delete_RR_output}
+    basedir, name = os.path.split(args.coadd)
+
+    if (args.filename_priors == 'to_change') | (args.filename_output_rerun_RR == 'to_change') | (args.filename_redrock_rerun_RR == 'to_change'):
```

Comment:
It seems strange to overwrite user-given inputs because another argument wasn't given (and wasn't required). That is what happens in your code if any of the three filenames isn't given explicitly. 

If I want two files named a certain way but don't care what you name the third, why shouldn't I be able to do that? Otherwise it seems like you should ask for one argument (perhaps some sort of name prefix/suffix/modifier) for all three files if you always treat them as a coupled set.

Reply:
Ok I changed the file ! I put only the default value when there is no entry.